index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8">
   <title>Fork Me! FCC: Test Suite Template</title>
   <link rel="stylesheet" href="css/style.css">
-</head>
+</head> 
 
 <body>
   <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- [ ] Try to use semantic tags. These divs could be changed to section tags.

Reply:
- [ ] What do you think if we try to make this comment a bit more polite and kind. We can start the comments with: "I think..." or "What if...". This way our comments sound less like orders, and more like a conversation with a peer.

Reply:
- [ ] I think it could be better if you use semantic tags here and change the divs into section tags =)